index.html

```diff
@@ -532,9 +532,9 @@ <h3>Proofs</h3>
 
         <p>
 When expressing a <a>data integrity proof</a> on an object, a
-<dfn class="lint-ignore">`proof`</dfn> property MUST be used where its value is a
-single object, or an unordered set of objects, expressed using the properties
-below:
+<dfn class="lint-ignore">`proof`</dfn> property MUST be used. If present, its
+value MUST be either a single object, or an unordered set of objects, expressed
```

Comment:
```suggestion
<dfn class="lint-ignore">`proof`</dfn> property MUST be used and its
value MUST be either a single object, or an unordered set of objects, expressed
```

Reply:
If you feel strongly about this, I'm fine w/ leaving it as is. I don't think the new wording changes the meaning of the sentence.

Reply:
I prefer as-is, because this doesn't spark the need for further review/rewording.